scout/server/blueprints/variants/templates/variants/indicators.html

```diff
@@ -51,6 +51,17 @@
   {% endif %} <!-- end of if variant.matching_tiered -->
 {% endmacro %}
 
+{% macro matching_manual_rank(variant) %}
+  {% if variant.matching_ranked %}
+    <span class="popovers badge bg-dark" data-bs-toggle="popover" data-bs-html="true" title=""
```

Comment:
The popover doesn't seem to work for me? Tested locally with the 2 RD demo cases

Reply:
Hm, I so convinced it worked before, but if I could have few bucks for each of those... 😄 Now it works, hopefully again. 😸 
![Screenshot 2023-01-31 at 11 15 13](https://user-images.githubusercontent.com/758570/215733289-8e327a4a-79dd-4ca5-9a1a-96abdf68538e.png)

Reply:
This could be a little confusing with new terms I guess. Here's a verbose version:
![Screenshot 2023-01-31 at 12 36 14](https://user-images.githubusercontent.com/758570/215749846-a5f17e74-ff1c-4392-b0a5-65bbe618e989.png)